pkgs/by-name/gr/graphene-hardened-malloc/package.nix

```diff
@@ -23,9 +23,17 @@ stdenv.mkDerivation (finalAttrs: {
   # these tests cover use as a build-time-linked library
   checkTarget = "test";
 
+  buildPhase = ''
+    runHook preBuild
+
+    for VARIANT in default light; do make $makeFlags ''${enableParallelBuilding:+-j$NIX_BUILD_CORES} VARIANT=$VARIANT; done
+
+    runHook postBuild
+  '';
```

Comment:
Please use makeTargets instead

Reply:
From my tinkering, looks like both `makeTargets` and `makeFlags` won't work with how hardened_malloc's Makefile handles VARIANT.

From what I experimented with, we have 2 variants left:
- `let mkMalloc = variant: stdenv.mkDerivation {}; in { graphene-hardened-malloc = mkMalloc "default"; graphene-hardened-malloc-light = mkMalloc "light"; }`
  but this will create `graphene-hardened-malloc.default` and `graphene-hardened-malloc.light` + I don't like how these `let blabla = stdenv.mkDerivation in` look 🙄 
- we can introduce an overridable variant attribute like [mimalloc package](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/libraries/mimalloc/default.nix#L2) and then override it in the module.